docs/features/writing-different-telemetry-types.md

```diff
@@ -32,6 +32,7 @@ We provide support for the following dependencies:
 
 - [HTTP](#measuring-http-dependencies)
 - [SQL](#measuring-sql-dependencies)
+- [Custom](#measuring-custom-dependencies)
```

Comment:
Can you sort these alpahbetically please?

Reply:
Done but left the actual contents as it is (?).